cli/src/lib/plugin/webpack.config.js

```diff
@@ -0,0 +1,316 @@
+// Based on CRA Webpack config
```

Comment:
Were there any significant changes you had to make to the CRA config?

Reply:
Also, should this maybe live in the /config directory with our other config files?

Reply:
Two main changes:
1. Handling of fonts and other non-js files
2. Modifying the 'InjectManifest' plugin

I tried to mark these with `// dhis2:` comments; in [this PR to incorporate the feedback](https://github.com/dhis2/app-platform/pull/773) I added one to line 279 🙂